app/views/admin/announcements/_announcement.html.haml

```diff
@@ -5,9 +5,11 @@
   .announcements-list__item__action-bar
     .announcements-list__item__meta
       - if announcement.scheduled_at.present?
-        = t('admin.announcements.scheduled_for', time: l(announcement.scheduled_at))
+        %time.formatted{ datetime: announcement.scheduled_at.iso8601, title: l(announcement.scheduled_at) }
+          = t('admin.announcements.scheduled_for', time: l(announcement.scheduled_at))
```

Comment:
The `admin.announcements.scheduled_for` (“Scheduled for %{time}”) is going to be overwritten by the JS code, and the “Scheduled for” part lost in the process.

Reply:
Good catch.

Removed this one from the PR, will do follow to find a way to keep JS integration/i18n in place.